index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/8f76becb1e.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,400&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    
+    <!-- header section start -->
+    <section class="header"> 
+        <!-- The navbar section  start-->
+        <nav class="nav-bar">
+            <h1 class="logo-txt">
+                <a href="#">My Logo</a>
+            </h1>
+            <i class="fa-solid fa-bars"></i>
+        </nav> 
+        <div class="container">
+            <h1>
+                I'm Benjamin 
+                Glad to see   you!
+            </h1>
```

Comment:
- [ ] OPTIONAL:  To stick to the Figma design, you can separate the two sentences into different` hx` tags. 
```suggestion
            <h1>I'm Benjamin</h1>
            <h2> Glad to see   you!</h2>
```

**checkout how it looks on some mobile screen sizes:**
![image](https://user-images.githubusercontent.com/80612925/176724670-f34d3153-9399-4308-a5e7-6389a58ee721.png)